src/SharedModule/components/NavBar/NavBar.tsx

```diff
@@ -1,7 +1,37 @@
-import React from 'react'
-
+import navLogo from "../../../assets/images/nav-logo.png"
 export default function NavBar() {
   return (
-    <div>NavBar</div>
+    <>
+    <nav className="navbar navbar-expand-lg navbar-light bg-light">
+  <div className="container-fluid">
+    <a className="navbar-brand" href="#">
+      <img src={navLogo} alt=""  />
```

Comment:
Consider adding alternative text for better accessibility and displaying the name in case of not being able to display the image.